gensim/scripts/word2vec2tensor.py

```diff
@@ -63,22 +64,25 @@ def word2vec2tensor(word2vec_model_path, tensor_filename, binary=False):
         True if input file in binary format.
 
     """
+
     model = gensim.models.KeyedVectors.load_word2vec_format(word2vec_model_path, binary=binary)
     outfiletsv = tensor_filename + '_tensor.tsv'
     outfiletsvmeta = tensor_filename + '_metadata.tsv'
 
-    with open(outfiletsv, 'w+') as file_vector:
-        with open(outfiletsvmeta, 'w+') as file_metadata:
+    with utils.smart_open(outfiletsv, 'wb') as file_vector:
+        with utils.smart_open(outfiletsvmeta, 'wb') as file_metadata:
             for word in model.index2word:
-                file_metadata.write(gensim.utils.to_utf8(word) + gensim.utils.to_utf8('\n'))
+                write = gensim.utils.any2utf8(word) + gensim.utils.any2utf8('\n')
```

Comment:
no needed changes (this is almost same), you can return an old variant (except for last line in cycle)